spec/controllers/resraurants_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, :type => :controller do
+  before(:all) do
+    @restaurants = FactoryBot.create_list(:restaurant, 3)
+  end
+
+  describe 'GET restaurants' do
+    it 'returns array with restaurants' do
+      get :index
+
+      expect(response.status).to eq(200)
+      expect(response.body).not_to be_empty
```

Comment:
This doesn't really test for anything, Even if the body is gibberish then it wouldn't be empty. I would at least validate its length is 3